src/components/Node/Node.js

```diff
@@ -97,6 +99,27 @@ export default memo(function Node({
     [node.data.simpleClaims]
   );
 
+  const occupation = useMemo(
+    () =>
+      renderOccupations(occupationByQid(node.data.simpleClaims[OCCUPATION_ID])),
```

Comment:
ok, but actually in next I moved these into the formatEntity so are readily available

Reply:
ok let's move it later